index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" type="text/css" href="main.css">
+        <title>Sing in project</title>
+    </head>
+
+    <body>
+        <div id="container">
+            <div id="col-1"></div>
+            <div id="col-2">
+                <div id="content-top">
+                    <div>
+                        <img src="./img/intuit_logo.jpg" alt="intuit" id="logo">
+                    </div>
+                    <div id="items">
+                        <ul>
+                            <li><a href="#"> <img src="./img/mint.jpg" alt=""></a> </li>
+                            <li><a href="#"><img src="./img/quickboks.jpg" alt=""></a></li>
+                            <li><a href="#"><img src="./img/turbotax.jpg" alt=""></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div id="form-container">
+                    <div>
+                        <div>
+                            <h1>Create an Intuit account</h1>
+                        </div>
+                        <div>
+                            <p id="txt-1">One account for averything Intuit, Including Ming.
+                        </div>
+                        <div>
+                            <a id="a-1" href="#">Learn more</a></p>
+                        </div>
+
+
+                    </div>
+
+                    <div>
+                        <form idaction="">
+                            <div>
+                                <label for="emailUser" id="input-lb-email">Email address</label>
+                                <div id="input-email">
+                                    <input type="text">
+                                </div>
+
+                            </div>
+                            <div>
+                                <label for="phone" id="input-lb-phone">Phone (recommended)</label>
+                                <div id="input-phone">
+                                    <input type="text">
+                                </div>
+                            </div>
+                            <div>
+                                <label for="password" id="input-lb-pass">Password</label>
+                                <div id="input-pass">
+                                    <input type="text">
```

Comment:
`REQUIRED`

Consider using `type="password"` for Password input, this makes adding custom validation and handling of password more convenient among other dedicated features.

-----
`OPTIONAL`

Please check [this comment]() on associating `<input>` and `<label>` element.